api.md

```diff
@@ -522,6 +522,7 @@ the same semantics as the `show_chunks` [function][show chunks].
 | `table_name` | Hypertable name from which to drop chunks. If not supplied, all hypertables are affected.
 | `schema_name` | Schema name of the hypertable from which to drop chunks. Defaults to `public`.
 | `cascade` | Boolean on whether to `CASCADE` the drop on chunks, therefore removing dependent objects on chunks to be removed. Defaults to `FALSE`.
+| `cascade_to_materializations` | Boolean on whether the data in continuous aggregates which are backed by the chunk should be deleted. Defaults to `NULL`. Currently cannot be set to `FALSE`.
```

Comment:
this sounds a bit complicated, maybe change to `Boolean to control whether deletes on chunks cascade to materializations`

Reply:
hmm, I think that's a little to "repeats exactly what the argument name is" for me 🤔

Reply:
also i think we try to avoid `currently` in docs, also why can it not be set to false if its a boolean, shouldnt the default be either true or false and NULL means it's whatever the default is

Reply:
it just reads weird, i like the style used below more: `(BOOLEAN) Set to true to drop objects dependent upon chunks being dropped. Defaults to false.`
but i'm not native speaker so i'll let others chime in

Reply:
Set to `TRUE` to delete chunk data in associate continuous aggregates. Defaults to `NULL`. `FALSE` is not yet supported.

Reply:
Can you elaborate on what this parameter does?